meteor/server/publications/_publications.ts

```diff
@@ -23,3 +24,26 @@ import './translationsBundles'
 import './triggeredActionsUI'
 import './mountedTriggers'
 import './deviceTriggersPreview'
+
+import { MeteorPubSub } from '../../lib/api/pubsub'
+import { CorelibPubSub } from '@sofie-automation/corelib/dist/pubsub'
+import { PeripheralDevicePubSub } from '@sofie-automation/shared-lib/dist/pubsub/peripheralDevice'
+import { MeteorPublications } from './lib'
+import { logger } from '../logging'
+
+// Ensure all the publications were registered at startup
+if (Meteor.isDevelopment) {
+	function checkPublications(pubNames: string[]) {
+		for (const pubName of pubNames) {
+			if (!MeteorPublications[pubName]) {
+				logger.error(`Publication "${pubName}" is not setup!`)
+			}
+		}
+	}
+
+	Meteor.startup(() => {
+		checkPublications(Object.values<string>(MeteorPubSub))
+		checkPublications(Object.values<string>(CorelibPubSub))
+		checkPublications(Object.values<string>(PeripheralDevicePubSub))
```

Comment:
This is slightly dangerous in the future, since someone might add another PubSub and forget to add it here.
Is there a way to expose some joined enum or list of values (like a `AllPubSub` from `lib/api/pubsub` and use that instead?